README.md

```diff
@@ -142,6 +142,10 @@ Please post your problem in [our free support forum](http://community.rtcamp.com
 
 ## Changelog ##
 
+### 2.2.2 ###
+* Fix ssue for 'Save All Changes' if localized string value sent for button on form submit. [#236](https://github.com/rtCamp/nginx-helper/pull/236) - by [umeshnevase](https://github.com/umeshnevase)
```

Comment:
Fix issue where settings not saved because the button's value localized (for any language). [#236](https://github.com/rtCamp/nginx-helper/issues/236) - by [umeshnevase](https://github.com/umeshnevase)